examples/lessons/11/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "luma.gl-examples-lesson11",
+  "description": "An es6 example for luma.gl.",
+  "contributors": [
+    "Jian Huang <jianh@uber.com>"
+  ],
+  "scripts": {
+    "start": "webpack-dev-server --progress --hot --open -d",
+    "start-local": "webpack-dev-server --env.local --progress --hot --open -d"
+  },
+  "dependencies": {
+    "luma.gl": ">=4.0.0",
+    "math.gl": "1.0.0-alpha.7"
```

Comment:
`>=1.0.0-alpha.7`, will pick up improved versions. Current form will pin it.

Reply:
done.